homeassistant/components/ads/light.py

```diff
@@ -60,18 +83,37 @@ def __init__(
         ads_hub: AdsHub,
         ads_var_enable: str,
         ads_var_brightness: str | None,
+        ads_var_color_temp_kelvin: str | None,
+        min_color_temp_kelvin: int | None,
+        max_color_temp_kelvin: int | None,
         name: str,
     ) -> None:
         """Initialize AdsLight entity."""
         super().__init__(ads_hub, name, ads_var_enable)
         self._state_dict[STATE_KEY_BRIGHTNESS] = None
+        self._state_dict[STATE_KEY_COLOR_TEMP_KELVIN] = None
         self._ads_var_brightness = ads_var_brightness
+        self._ads_var_color_temp_kelvin = ads_var_color_temp_kelvin
+        self._min_color_temp_kelvin = min_color_temp_kelvin
+        self._max_color_temp_kelvin = max_color_temp_kelvin
```

Comment:
I think you don't use these two attributes, so it would be a good idea to remove them.